A32NX/html_ui/Pages/VCockpit/Instruments/A320_Neo/Map/Svg/A32NX_SvgConstraintElement.js

```diff
@@ -0,0 +1,61 @@
+class A32NXSvgConstraintElement extends SvgConstraintElement {
+    createDraw(map) {
+        const fontSize = map.config.waypointLabelFontSize;
+        let text = "CSTR";
+        let speedText = "";
+        if (this.source) {
+            if (this.source.legAltitude1 >= 10000) {
+                text = "FL" + (this.source.legAltitude1 / 100).toFixed(0);
```

Comment:
You should take into account the transition altitude. When the constraint is above the transition altitude, output `FLxxx`.

Reply:
Thanks for the feedback, I'm pretty new to aviation, can you please explain the rule that needs to follow in order to output FL100 instead of 10000 feet for example?
So If I understand correctly FL is tied to the transition altitude? if we're below transition altitude we need to show 12000 feet instead of FL120 for a transition altitude of 18000?

Reply:
Please read [this article](https://www.skybrary.aero/index.php/Transition_Altitude/Level) to learn about transition altitude and transition level. Your understanding is correct. At or below the transition altitude, all altitudes are fully written out (or communicated when talking to ATC). Above the transition altitude flight level is used.

Reply:
Should be taking Transition Altitude into account now, thanks for the feedback 😄

Reply:
Getting there! All that's left is getting the SimVar value using `SimVar.GetSimVarValue`, as the transition altitude can be changed via the MCDU.

Reply:
Already done in [here](https://github.com/flybywiresim/a32nx/pull/1812/files#diff-85c33a591b07d9a6297dea2ce064f26463ed1e8fee48aaf14a646dd33ec39046R518)